File: myCS/codeWarsKatas/Credit Card Mask/kata.cs
using System;

public static class Kata
{
  public static string Maskify(string cc)
  {
    if (cc.Length <= 4)
      return cc;

      char[] char_cc = cc.ToCharArray();
      Array.Reverse(char_cc);

      for (int i = 0; i < cc.Length; ++i){
        if (i >=4)
        char_cc[i] = '#';
      }

      Array.Reverse(char_cc);
      return new string (char_cc);
  }
}

File: myJava/codeWarsKatas/Count characters in your string/Kata.java
import java.util.Map;
import java.util.HashMap;

public class Kata {
    public static Map<Character, Integer> count(String str) {
      Map<Character, Integer> countMap = new HashMap<Character, Integer>();
      
      for (char i : str.toCharArray()) {
        if(!countMap.containsKey(i)) {
          countMap.put(i, 1);
        } else {
          countMap.replace(i, countMap.get(i) + 1);
        }
      }
      
      return countMap;
    }
}